includes/classes/class-plugin-manager.php

```diff
@@ -0,0 +1,568 @@
+<?php
+/**
+ * Class PerformanceLab\Plugin_Manager
+ *
+ * @package PerformanceLab
+ * Author: WordPress Performance Team
+ * Author URI: https://make.wordpress.org/performance/
+ * License: GPLv2 or later
+ * License URI: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ */
+
+namespace PerformanceLab\classes;
+
+use WP_Error;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * General purpose class for managing installation/activation of standalone performance plugins
+ * and rendering of the UI associated therewith.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Plugin_Manager {
+
+	/**
+	 * Get info about all the standalone plugins and their status.
+	 *
+	 * @return array of plugins info.
+	 */
+	public static function get_standalone_plugins() {
+		$managed_standalone_plugins = array(
+			'webp-uploads'                => array(
+				'Name'        => esc_html__( 'WebP Uploads', 'performance-lab' ),
+				'Description' => esc_html__( 'This plugin adds WebP support for media uploads within the WordPress application.', 'performance-lab' ),
+				'Author'      => esc_html__( 'WordPress Performance Team', 'performance-lab' ),
+				'AuthorURI'   => esc_url( 'https://make.wordpress.org/performance/' ),
+				'PluginURI'   => esc_url( 'https://wordpress.org/plugins/webp-uploads/' ),
+				'Download'    => 'wporg',
+			),
+			'sqlite-database-integration' => array(
+				'Name'        => esc_html__( 'SQLite Database Integration', 'performance-lab' ),
+				'Description' => esc_html__( 'Allows testing an SQLite integration with WordPress and gather feedback, with the goal of eventually landing it in WordPress core.', 'performance-lab' ),
+				'Author'      => esc_html__( 'WordPress Performance Team', 'performance-lab' ),
+				'AuthorURI'   => esc_url( 'https://make.wordpress.org/performance/' ),
+				'PluginURI'   => esc_url( 'https://wordpress.org/plugins/sqlite-database-integration/' ),
+				'Download'    => 'wporg',
+			),
+		);
```

Comment:
All this shouldn't be hard-coded. Instead, we should rely on the `plugin.json` file for the slugs of which plugins to cover here, and all other data should come from the WordPress.org API, which we can query in the same way that it's done on the _Plugins > Add New_ screen.